translations/ru_RU.ts

```diff
@@ -1514,11 +1514,12 @@ If it&apos;s partially checked, the global setting in Code Edit will be used.</s
     </message>
     <message>
         <source>YAPF</source>
```

Comment:
Can we remove this word from translations.
YAPF is YAPF.

Reply:
You can simply leave it blank and mark it finished.